vendor.conf

```diff
@@ -161,6 +161,6 @@ github.com/morikuni/aec                             39771216ff4c63d11f5e604076f9
 # metrics
 github.com/docker/go-metrics                        d466d4f6fd960e01820085bd7e1a24426ee7ef18
 
-github.com/opencontainers/selinux                   3a1f366feb7aecbf7a0e71ac4cea88b31597de9e # v1.2.2
+github.com/opencontainers/selinux                   5215b1806f52b1fcc2070a8826c542c9d33cd3cf
```

Comment:
@thaJeztah where does this change come from?

Reply:
This comes from https://github.com/moby/moby/pull/40404

Reply:
Ok, so it's more like when we vendored runc v1.0.0-rc9 we forgot to bump selinux, because there is no difference between v1.0.0-rc9 and v1.0.0-rc10 wrt selinux.